### DISCUSSION TITLE
compatibility

### DISCUSSION THREAD

**Discussion by emersonart** [kreait/firebase-php#652] [Ideas]
*** I'm translating on google translate from Brazilian Portuguese to English 

Would it be possible to adjust the library in the most updated version to be compatible with previous versions of php? at least version 7.2, currently only supports 7.4+

Some servers do not have newer versions yet.

(Yes, I know I could use a previous version of the library, but I would miss updates from the firebase itself)

This library helped me a lot!!! thanks!

**Comment by jeromegamez** [comment]
Hey there 👋, I'm happy that you like the library and that it's useful to you!

Unfortunately, although I understand your requirement, my limited time and resources only allow me to support PHP versions that are themselves supported. Ensuring support for older PHP versions would take time that I can better spend on evolving the SDK.

I know this is not the answer you were hoping for, but I hope you understand! 🙏
